libs/web3/src/lib/use-ethereum-transaction-store.tsx

```diff
@@ -27,7 +27,13 @@ export interface EthStoredTxState extends EthTxState {
   methodName: ContractMethod;
   args: string[];
   requiredConfirmations: number;
-  requiresConfirmation: boolean; // whether or not the tx needs external confirmation (IE from a subscription even)
+  // whether or not the tx needs external confirmation (IE from a subscription even)
+  requiresConfirmation: boolean;
+  // whether or not to notify via toast
+  // true = force open toast
+  // false = force close toast
+  // undefined = leave alone
+  notify: boolean | undefined;
```

Comment:
We should be able to make it work as expected without introducing new prop. dialogOpen property toast visibility value.
We could add third argument to update function `keepDismissed?: boolean`
add condition
```
if (!keepDismissed) {
  transaction.dialogOpen = true;
}
```

than just use third parameter in use ethereum transaction manager in only one place
```
if (requiresConfirmation) {
  update(transaction.id, { status: EthTxStatus.Complete, receipt }, true);
}
```